js/plugins/conviva.js

```diff
@@ -283,6 +289,18 @@ var ConvivaAnalyticsPlugin = function(framework)
   };
 
   /**
+   * Validates the custom metadata passed in via page level settings.
+   * @private
+   * @method ConvivaAnalyticsPlugin#validateCustomMetadata
+   * @param  {object} metadata The Conviva custom metadata to validate
+   * @returns {boolean} true if valid, false otherwise
+   */
+  var validateCustomMetadata = function(metadata)
+  {
+    return !!metadata && _.isObject(metadata);
```

Comment:
Ah, did not know that. I knew that typeof null is object, but Underscore seems to take that into consideration. I'll make this change.